enclave-tls/src/Makefile

```diff
@@ -11,27 +11,42 @@ App_C_Files :=  $(wildcard $(LIBDIR)/core/*.c) \
 	$(wildcard $(ENCLAVE_QUOTES)/api/*.c $(ENCLAVE_QUOTES)/internal/*.c) \
 	$(wildcard $(TLS_WRAPPERS)/api/*.c $(TLS_WRAPPERS)/internal/*.c) \
 	$(wildcard $(LIBDIR)/api/*.c)
+
```

Comment:
The line no 3 "include ../common.mk" should be changed to "include $(TOPDIR)/common.mk".